Dockerfile

```diff
@@ -1,10 +1,46 @@
-FROM node:5
+FROM node:boron
```

Comment:
@mattclark this is the same Dockerfile I updated with ember-preprints. node:boron is the 6.X LTS release.